test/Microsoft.AspNet.Mvc.ModelBinding.Test/Validation/DataAnnotationsModelValidatorTest.cs

```diff
@@ -220,55 +220,6 @@ public void ValidateReturnsMemberNameIfItIsDifferentFromDisplayName()
 #endif
 
         [Fact]
-        public void GetClientValidationRules_ReturnsEmptyRuleSet()
```

Comment:
There are no replacement tests for these, the `DataAnnotationsClientModelValidator` is no longer there. The remaining class `DataAnnotationsClientModelValidator<T>` is abstract and the functionality is well tested by individual adapters.